pkg/operator/webhooksupportabilitycontroller/degraded_webhook.go

```diff
@@ -130,9 +139,13 @@ func (c *webhookSupportabilityController) assertConnect(ctx context.Context, web
 		var conn net.Conn
 		conn, err = dialer.DialContext(ctx, "tcp", net.JoinHostPort(host, port))
 		if err != nil {
+			err = fmt.Errorf("failed to connect to webhook %q via service %q: %v", webhookName, net.JoinHostPort(host, port), err)
+			if caBundleProvidedByServiceCA && len(caBundle) == 0 {
+				err = fmt.Errorf("%v. NOTE that the caBundle is provided by the service-ca-operator but was empty. Please check the service-ca's logs", err)
+			}
```

Comment:
I think we should perform this check before attempting the connection. There is little chance that the connection would succeed anyway. This would allow for a simpler error message for the status and less noise in the webhook logs for failed connections that would never have succeeded without the CA bundle.

Reply:
OK, I've updated the PR, PTAL.